README.md

```diff
@@ -160,6 +164,23 @@ aws iam delete-access-key --user-name=YOUR_ACCOUNT_NAME --access-key-id=YOUR_ACC
 
 To sync a subfolder in S3, add it like you would add any other file path. For example to sync all files in `foo` subfolder of the S3 bucket named `bar-bucket`, set `file_path==bar-bucket/foo`.
 
+### Parquet Partitioning
```

Comment:
Too much customization in config? It's getting pretty close to just writing code at this point, but it seems undesirable not to have the ability to filter by partition, and if we're doing that, it seems undesirable not to expose the full features of the underlying API.

Reply:
Removed parquet partitioning and filters.